src/app/app.component.html

```diff
@@ -0,0 +1,40 @@
+<nav>
```

Comment:
Doesn't make much difference actually. I believe that it's a common practice to name it the same as a ts file. But anyway in this project we can choose another way of doing it.